doofinder-for-woocommerce/includes/class-setup-wizard.php

```diff
@@ -427,10 +441,22 @@ public function getToken()
 	 */
 	public function getReturnPath()
 	{
+
+		/*
+		$setup_wizard_url = get_site_url(null, 'wp-content/plugins/doofinder-for-woocommerce/api/connect/');
+		$blog_id = get_current_blog_id();
+		$setup_wizard_url .= $blog_id === 1 ? "": "?site_id=".$blog_id;
+		*/
+
 		$setup_wizard_url = get_site_url(null, 'wp-json/doofinder-for-wc/v1/connect/');
 		return $setup_wizard_url;
 	}
 
+	public function getAdminPath()
+	{
+		return  "https://app.doofinder.com";
+	}

```

Comment:
This would be better stored in a const (exportable if you need to)

Reply:
I've moved the value into a class constant